documentation/modules/ROOT/pages/basics.adoc

```diff
@@ -97,3 +97,34 @@ Let's start the _Live Coding_ mode by invoking `mvn`. You probably won't need to
 ----
 ./mvnw quarkus:dev
 ----
+
+== Continuous Testing (Development mode)
+
+Quarkus supports continuous testing, meaning that your tests will run immediately after code changes were saved.
+Let's start again in _Live Coding_ mode by invoking `mvn`:
+
+[.console-input]
+[source,bash]
+----
+mvn quarkus:dev
+----
+
+Quarkus will start in development mode as normal, but down the bottom of the screen you should see the following:
+[.console-input]
+[source,bash]
+----
+Tests paused
+Press [r] to resume testing, [o] Toggle test output, [h] for more options>
+----
+
+Press `r` and the tests will start running. You should see the status change down the bottom of the screen as they are running, and it should finish similar to this:
+
+[.console-input]
+[source,bash]
+----
+All 1 test is passing (0 skipped), 1 test was run in 11705ms. Tests completed at 16:02:32.
+----
+
```

Comment:
Maybe I will add a change on the REST API to validate that the tests are automatically run and they can fail too.